packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -60,7 +59,14 @@ export const NavUser = ({ children, initials, ...props }: NavUserProps) => {
   };
 
   return (
-    <Flex justifyContent="center" {...props}>
+    <Flex
+      justifyContent="center"
+      {...props}
```

Comment:
I think the spread props should probably be last. It's usually a way to let the place calling the component (the place that does <NavUser />) override the default styles of the component